benchmark/hdf5/benchmark_knowhere_float_qps.cpp

```diff
@@ -194,7 +194,12 @@ class Benchmark_knowhere_float_qps : public Benchmark_knowhere, public ::testing
 
 TEST_F(Benchmark_knowhere_float_qps, TEST_IVF_FLAT_NM) {
 #ifdef USE_CUDA
+#ifdef KNOWHERE_WITH_RAFT
+    index_type_ = knowhere::IndexEnum::INDEX_RAFT_IVFFLAT;
+#else
+
     index_type_ = knowhere::IndexEnum::INDEX_FAISS_GPU_IVFFLAT;
```

Comment:
Do we possibly do all benchmarks together, enable RFAT does not mean FAISS_GPU is not possible, and also enabling GPU not means CPU is not possible.
Maybe another PR could handle it.

Reply:
@matrixji in the future work, we will Abandon support for GPU in Faiss. raft looks better.